src/main/scala/org/broadinstitute/dsde/agora/server/webservice/ApiServiceActor.scala

```diff
@@ -72,7 +72,8 @@ class ApiServiceActor(permissionsDataSource: PermissionsDataSource, healthMonito
   def withResourceFileContents(path: String)(innerRoute: String => Route): Route =
     innerRoute( FileUtils.readAllTextFromResource(path) )
 
-  def possibleRoutes =  options{ complete(OK) } ~ statusService.routes ~ ga4ghService.routes ~
+  // TODO Add statusService.routes back in
+  def possibleRoutes =  options{ complete(OK) } ~ ga4ghService.routes ~
```

Comment:
So, I'm having a hard time grokking the implementation. Current (and old) architecture is that spray runs *spray* routes. In the status endpoint construction below, we have a differently typed *akka* route. How do we make `ApiServiceActor` smart enough to be able to run both kinds of routes? I've played around with this a little and I don't see a technically feasible solution. They seem fundamentally incompatible.

Reply:
We won't be running both kinds of routes simultaneously. Removing `statusService.routes` temporarily is a stop-gap until the other routes are updated. I'll make the changes next for `possibleRoutes` to only include `statusService.routes` so it can still compile and we can test the converted route on Swagger. Does that make sense?

Reply:
We won't be running both kinds of routes simultaneously. Removing `statusService.routes` temporarily is a stop-gap until the other routes are updated. I'll make the changes next for `possibleRoutes` to only include `statusService.routes` so it can still compile and we can test the converted route on Swagger. Does that make sense?

Reply:
Ok, yes, that makes more sense. If we turn off `spray` and serve the app the `akka` way, that makes more sense. Then we could migrate route after route.

Reply:
This would only be valid as a *very* temporary stopgap. If all we change is the content of `possibleRoutes`, then we're running akka-http routes via spray-can (see `ServerInitializer.startWebServiceActors`), which isn't where we'll be in the end.

Reply:
Right. I'd meant fully removing Spray dependency from ApiServiceActor. Let me pull on that string and see how far it might go.

Reply:
sounds good! This would be a great place to have parallel methods - `startAkkaRoutes` and `startSprayRoutes` to make it easy to switch back and forth during development.